<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>569</width>
    <height>410</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QSlider" name="Slider_Angle">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>210</y>
      <width>160</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-30</number>
    </property>
    <property name="maximum">
     <number>30</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="Box_Angle">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>210</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-30</number>
    </property>
    <property name="maximum">
     <number>30</number>
    </property>
   </widget>
   <widget class="QPushButton" name="PB_open">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>160</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>OPEN</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="TB_file_path">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>160</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
